### PR TITLE
doc: update the man page entries for DirAuthority and FallbackDir

### PR DESCRIPTION
**Pull request by teor2345**
Improve the documentation for the DirAuthority and FallbackDir torrc
options.

Closes ticket 30955.

### CODE REVIEW

File: doc/tor.1.txt [hunk 1]
```diff
@@ -463,30 +468,36 @@ GENERAL OPTIONS
...
-    clients ignore this in favor of the specified "orport=" value.  If an
-    IPv6 ORPort is supplied, Tor will
-    also download directory documents at the IPv6 ORPort. +
+    download directory documents. Clients always use the ORPort. Relays
+    usually use the DirPort, but will use the ORPort in some circumstances.
+    If an IPv6 ORPort is supplied, clients will also download directory
+    documents at the IPv6 ORPort, if they are configured to use IPv6. +
+ +
+    Directory authorities should be stable relays with the same IP addresses,
+    ports, and identity keys. They must have a DirPort. +
```
Comment (nmathewson) [inline doc/tor.1.txt]: This paragraph appears twice.  I'm not sure it need to appear at all, though: users need to know a lot more than this if they want to set up a nonstandard set of directory authorities.

Also, the phrase "the same" should probably read "stable" or "unchanging" or something.

File: doc/tor.1.txt [hunk 2]
```diff
@@ -447,13 +447,18 @@ GENERAL OPTIONS
...
+[[FallbackDir]] **FallbackDir** __ipv4address__:__dirport__ orport=__orport__ id=__fingerprint__ [weight=__num__] [ipv6=**[**__ipv6address__**]**:__orport__]::
+    When tor is unable to connect to any directory cache for directory info
+    (usually because it doesn't know about any yet) it tries a hard-coded
+    directory. Relays try one directory authority at a time. Clients try
+    multiple directory authorities and FallbackDirs, to avoid hangs on
+    startup if a hard-coded directory is down. Clients wait for a few seconds
+    between each attempt, and retry FallbackDirs more often than directory
+    authorities, to reduce the load on the directory authorities.  +
+ +
+    FallbackDirs should be stable relays with the same IP addresses, ports,
```
Comment (nmathewson) [inline doc/tor.1.txt]: The phrase "the same" here probably should be something like "fixed" or "stable" or "unchanging".

Comment (teor2345) [reply doc/tor.1.txt]: I changed it to "stable" in a fixup.

File: doc/tor.1.txt (hunk 1 above)
Comment (teor2345) [reply doc/tor.1.txt]: I deleted both copies of this paragraph in a fixup.

**Comment by teor2345** [comment]
Merged in trac 30955.

**Pull request closed without merging**